### DISCUSSION TITLE
WOW Real-ESRGAN is AMAZING! thank YOU!

### DISCUSSION THREAD

**Discussion by brunoais** [xinntao/Real-ESRGAN#108] [General]
You saved me from competition such as https://vanceai.com/ or even https://www.topazlabs.com/denoise-ai
Your work wipes the floor with those commercial options!
Thank you very much for doing this and make it open source thank you!!!

**Comment by xinntao** [comment]
Thanks 😃 

Real-ESRGAN still cannot well handle many many real images. Hope we can continuously improve it 😸

**Reply by brunoais** [reply]
Even though it still has faults, the results are still astonishing!
Mostly how much it completely destroys closed-source competition.
It's impressive!

**Comment by suucss** [comment]
thank you, author,it is you that let me see the HD pictures, with open source and free of charge ,I am able to upscale my favorite picturess and share these with my friends ,how perfect the software is! thanks again!

**Comment by FlM122** [comment]
which model is the best for anime videos??? to ensure that the quality is as good as possible and that the speed is high . I found AnimeJaNai, but it makes the contour lines too sharp.
